src/Tools/Source/CompilerGeneratorTools/Source/BoundTreeGenerator/Model.cs

```diff
@@ -33,9 +59,21 @@ public class TreeType
 
         [XmlAttribute]
         public string HasValidate;
+
+        [XmlElement(ElementName = "TypeComment", Type = typeof(CommentNode))]
+        public List<CommentNode> Comments;
+
+        ref List<CommentNode> ICommentedNode.GetCommentListField()
+        {
+            return ref Comments;
+        }
+    }
+
+    public sealed class PredefinedNode : TreeType
+    {
     }
 
-    public class PredefinedNode : TreeType
+    public sealed class ValueType : TreeType
```

Comment:
>public sealed class ValueType : TreeType [](http://example.com/codeflow?start=4&length=40)

Consider leaving the type where it used to be. That will make the nature of the change clearer, thus it will be easier to follow it